### DISCUSSION TITLE
Count element in Webdriver.io

### DISCUSSION THREAD

**Discussion by QuocBaoNguyen11295** [webdriverio/webdriverio#9774] [Community]
Hi community
I have a following DOM: 
```
<select id="selectbox">
    <option value="someValue0">uno</option>
    <option value="someValue1">dos</option>
    <option value="someValue2">tres</option>
    <option value="someValue3">cuatro</option>
    <option value="someValue4">cinco</option>
    <option value="someValue5">seis</option>
</select>
```

I would like to count the amount of option tag
How can I count option tag? or do we have any functionality in Webdriver.io for counting elements?
Thank you community

**Comment by christian-bromann** [comment]
How about: `const cnt = await $$("select option").length`

**Reply by QuocBaoNguyen11295** [reply]
Thank you @christian-bromann 
It works for me
